erpnext/accounts/doctype/payment_entry/payment_entry.js

```diff
@@ -886,20 +886,7 @@ frappe.ui.form.on('Payment Entry', {
 			var allocated_positive_outstanding =  paid_amount + allocated_negative_outstanding;
 		} else if (["Customer", "Supplier"].includes(frm.doc.party_type)) {
 			total_negative_outstanding = flt(total_negative_outstanding, precision("outstanding_amount"))
-			if(paid_amount > total_negative_outstanding) {
-				if(total_negative_outstanding == 0) {
-					frappe.msgprint(
-						__("Cannot {0} {1} {2} without any negative outstanding invoice", [frm.doc.payment_type,
-							(frm.doc.party_type=="Customer" ? "to" : "from"), frm.doc.party_type])
-					);
-					return false
-				} else {
-					frappe.msgprint(
-						__("Paid Amount cannot be greater than total negative outstanding amount {0}", [total_negative_outstanding])
-					);
-					return false;
-				}
-			} else {
+			if(paid_amount < total_negative_outstanding) {
```

Comment:
@barredterra I think it makes sense to reverse the original if condition since we're removing the path that triggered the validation.  I haven't spent much time reading this section of PE logic. Do you have any initial doubts about this?